code/modules/global_listener/interfaces.dm

```diff
@@ -42,6 +42,19 @@ Iterating through a listener list:
 	target = null
 	return ..()
 
+//-------------------------------
+// Record listener
+//-------------------------------
+
+/listener/record/New(datum/target)
```

Comment:
What is this used for?

Reply:
This is used as general record changes listener. It's supposed to be overridden to be able to handle differently interactions for different target listeners...
_Also I could have just added global methods and just called them, but I am not sure about that_